.github/workflows/interop-tests.yml

```diff
@@ -0,0 +1,50 @@
+name: Interoperability tests with GnuTLS and NSS
+on: [pull_request, push]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 90
+    strategy:
+      fail-fast: false # Setting fail-fast to false prevents GitHub from cancelling all in-progress jobs if any matrix job fails.
+      matrix:
+        COMPONENT: [openssl-upstream-gnutls, openssl-upstream-nss]
+    env:
+      COMPONENT: ${{ matrix.COMPONENT }}
+    # Code below inspired by https://fedoramagazine.org/github-actions-use-podman-to-run-fedora-linux/
+    steps:
+      - name: Setup Podman
+        run: |
+          sudo apt -y update
+          sudo apt-get -y install podman
+          echo "podman version"
+          podman version
+          podman pull registry.fedoraproject.org/fedora:latest
+      # - name: Get source
+      #   uses: actions/checkout@v3
+      #   with:
+      #     submodules: true
+      - name: Create container
+        run: |
+          {
+              echo 'FROM fedora:latest'
+              echo 'RUN dnf -y update'
+              echo 'RUN dnf -y install perl'
+              echo 'RUN dnf -y install git tmt-all'
+              echo 'RUN dnf -y remove openssl'
+              echo 'RUN mkdir /root/bin'
+              echo "RUN echo '#!/bin/bash /openssl/util/wrap.pl /openssl/apps/openssl' > /root/bin/openssl & chmod +x /root/bin/openssl"
+              echo 'RUN git clone -b interop-plans --recurse-submodules https://github.com/not4pedro/openssl.git'
+              echo 'WORKDIR /openssl/'
+              echo 'RUN ./config --banner=Configured no-asm no-makedepend enable-buildtest-c++ enable-fips --strict-warnings -D_DEFAULT_SOURCE && perl configdata.pm --dump'
```

Comment:
What is the reason for `-D_DEFAULT_SOURCE` here?

I'd drop `enable-buildtest-c++` and possibly `no-asm` (less sure about this, it depends on what you are trying to test).